### PR TITLE
v0.1.0 charts for CrateDB, Orion and QuantumLeap

### PR DESCRIPTION
**Pull request by chicco785**

### CODE REVIEW

File: crate/README.MD [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* PV support on the underlying infrastructure
+
+## StatefulSet Details
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/
+
+## StatefulSet Caveats
+* https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#limitations
+
+## Todo
+*
```
Comment (taliaga) [inline crate/README.MD]: let's add: follow up with https://github.com/helm/charts/pull/8432

File: orion/README.MD [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# CrateDB Helm Chart
```
Comment (taliaga) [inline orion/README.MD]: fix: Orion

File: orion/README.MD [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+| `image.tag`                       | The version of the image to pull            | `2.2.0`                                             |
+| `image.pullPolicy`                | The pull policy                             | `IfNotPresent`                                      |
+| `resources`                       | Any resources you wish to assign to the pod | `{}`                                                |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+$ helm install --name my-release -f values.yaml oc/crate
```
Comment (taliaga) [inline orion/README.MD]: fix: orion

File: crate/README.MD [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+To install the chart with the release name `my-release`:
+
+```console
+$ helm repo add oc https://orchestracities.github.io/charts/
+$ helm dependency update
+$ helm install --name my-release oc/crate
+```
+
+## Connecting to CrateDB
+
```
Comment (taliaga) [inline crate/README.MD]: what's planned here? something like port-forward instructions or execs into pod maybe?

Comment (chicco785) [reply crate/README.MD]: yes!

File: crate/README.MD (hunk 1 above)
Comment (chicco785) [reply crate/README.MD]: didn't recall about it... not yet approved... pf :( if not we could just remove this chart

Comment (taliaga) [reply crate/README.MD]: Yeap, a ci issue no-one could help me with. looked like a time-out thing also happening to other people. Sure, we can come back to that at some point and when merged remove this one from here.

File: crate/README.MD (hunk 4 above)
Comment (taliaga) [reply crate/README.MD]: ok, maybe we don't need to write this for all charts in the readme. There was a part in the templates where this was actually written so it'd be printed once the chart is deployed. Less text to maintain I mean.

**Comment by kfox1111** [comment]
Could these charts be added to https://hub.helm.sh/ ?

**Comment by kfox1111** [comment]
No worries.

The details are here I think:
https://github.com/helm/hub/blob/master/README.md

**Pull request merged**